benchmarks/scala-stdlib/build.sbt

```diff
@@ -1,5 +1,9 @@
 lazy val renaissanceCore = RootProject(uri("../../renaissance-core"))
 
+lazy val scalaSMTLib = RootProject(uri("scala-smtlib"))
+
+lazy val scalaCafeSAT = RootProject(uri("cafesat"))
```

Comment:
IMO, we should open a new subproject for benchmark, such as `cafesat`, and not pollute `scala-stdlib` with this.
At some point, we will potentially transition to a newer Scala version in `scala-stdlib`, and at that point this will make migration harder.

Reply:
You mean a new group altogether containing sub projects for smt and cafesat and the doku benchmark?

Reply:
Yes.

Reply:
Will do.